tests/acceptance/customCommands/filesListScrollToTop.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Scroll the files list to the beginning
+ */
+exports.command = async function () {
+  await this.executeAsync(function (done) {
+    const filesListScroll = document.querySelector('#files-list-container')
```

Comment:
might be better to make this generic "scrollElementToTop" and get the selector as argument